src/XRP/default-xrp-client.ts

```diff
@@ -322,11 +322,7 @@ export default class DefaultXrpClient implements XrpClientDecorator {
             getTransactionResponse,
             this.network,
           )
-          if (!xrpTransaction) {
-            throw XrpError.paymentConversionFailure
-          } else {
-            payments.push(xrpTransaction)
-          }
+          payments.push(xrpTransaction)
```

Comment:
is this change related? it seems reasonable but can't tell

Reply:
It's a result of `XrpTransaction.from()` not being able to return undefined anymore.